src/components/TsvContent/TsvContent.js

```diff
@@ -219,8 +220,10 @@ const Item = ({
               margin: markdownView ? '10px 0px 0px' : '-5px 0px 0px',
             }}
             onEdit={markdown => {
-              setUpdatedItem('markdown', markdown)
-              onTsvEdit({ [markdownLabel]: markdown })
+              console.log('markdown', markdown)
+              console.log('clean markdown', cleanMarkdownLineBreak(markdown))
```

Comment:
Maybe remove the logs?

Reply:
Oh yeah! Good catch. I totally forgot about those.